guides/source/getting_started.md

```diff
@@ -92,6 +92,10 @@ The version shown should be Rails 8.1.0 or higher.
 
 ### Creating Your First Rails App
 
+Note: In this guide, while specifying commands we use `rails` or `bin/rails`.
+Both commands are correct. `rails` will always delegate to `bin/rails`. But
+we should use commands from the `bin/` folder when running them within an application.
+
```

Comment:
When running `rails` outside a Rails application it won't delegate to `bin/rails`.
So if we keep the comment here, it should be something like:
```suggestion
Note: When generating a new Rails application we use `rails`.
But we should run commands  with `bin/rails` when running them within an application.

```

I think this should be moved to the first occurrence of `bin/rails`  though.